### ISSUE TITLE
Changing premade keyboard shortucts?

### ISSUE THREAD

**Issue by peter099** [NickSto/websearchpro#8]
Is it possible to change the premade keyboard shortuct that invoke specific search engine? Or add new keyboard shortcuts?

**Comment by MartijnKooij** [comment]
You can edit the shortcuts for all search engines in the options of WSPro. On the "Search Engines" pane, in the list of your search engines there's an "Edit" button which will popup an edit screen in which you can also edit the shortcut

**Comment by peter099** [comment]
Yes, but i only see a list of premade shortcuts, i don't see a option to add new shortcuts.

**Comment by MartijnKooij** [comment]
Ah OK. No, that is currently not possible so that would be a feature request. Not sure if new features will be added though, we'll have to ask Nick about that.
